web/components/home/Home.js

```diff
@@ -176,17 +176,23 @@ class Home extends React.Component {
                         className="scale-width-mobile"
                       />
                     </Link>
-                    {/** <a
-                      target="_blank"
-                      href="https://docs.google.com/forms/d/1HQDZ2YWv8NCkSznOf-Bp6b14gRENiqoSCWYQ8_rYFLM/edit?usp=drivesdk"
-                      rel="noopener noreferrer"
-                      className="scale-width-mobile"
+                    <Link to="/application/volunteer" className="scale-width-mobile margin-right-s scale-margin-sides-mobile-none scale-margin-bottom-mobile-s">
+                      <SecondaryButton
+                        text="Become a volunteer"
+                        className="scale-width-mobile"
+                      />
+                    </Link>
+                    <a
+                    target="_blank"
```

Comment:
The travis build is failing, you can see the output by clicking [Details](https://github.com/nwplus/nwhacks2019/pull/189/checks?check_run_id=33221059) and then "the build" on that page. According to the long, the lint is failing:

https://travis-ci.com/nwplus/nwhacks2019/builds/91946198#L510-L514

You can also run `make lint` locally to find the errors, which is a lot quicker than waiting for travis. In this case, you just need to indent `L186 - L189` and the build should pass.

https://github.com/nwplus/nwhacks2019/blob/44ea4ad853a1e12e859dafc8bc3ebfdffdfb673d/web/components/home/Home.js#L185-L190

Reply:
thanks!